service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ObservationValueQuantityMapper.java

```diff
@@ -68,9 +72,9 @@ private static String getPhysicalQuantityXml(Quantity valueQuantity) {
                 .formatted(
                     valueQuantity.getValue(),
                     valueQuantity.getValue(),
-                    valueQuantity.getCode(),
+                    escapedCode,
                     getSystemWithoutPrefix(valueQuantity.getSystem()),
```

Comment:
Should we be worried about the system here at all? It's a string.

Reply:
[This method](https://github.com/NHSDigital/integration-adaptor-gp2gp/blob/0d00ec63f4b9accd1af7b13e1da5ed5dccef7683/service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ObservationValueQuantityMapper.java#L213) validates the system before it is used.  So if it isn't specifically `http://unitsofmeasure.org`, an OID, or a URN, then it isn't used.  As these don't contain XML escape characters we don't need to escape it.